crdppf/views/printproxy.py

```diff
@@ -113,9 +113,15 @@ def status(self):
     def report_get(self):
         """ Get the PDF. """
 
-        return self._proxy_response(
+        pdf = self._proxy_response(
             "%s/report/%s" % (
                 self.config['print_url'],
                 self.request.matchdict.get('ref')
             ),
         )
+        filename = pdf.content_disposition.split('=')[1]
+        with open('C:/Temp/'+filename, 'wb') as f:

```

Comment:
I am sure that you are planning to modify this line...

Reply:
yes I do (see description) 😄